weblate/addons/management/commands/install_addon.py

```diff
@@ -67,7 +67,8 @@ def handle(self, *args, **options):
             addons = Addon.objects.filter_component(component).filter(name=addon.name)
             if addons.exists():
```

Comment:
Evaluate the query here as we will need all the objects in case they exist anyway:

```suggestion
            if addons:
```